leshan-integration-tests/src/test/java/org/eclipse/leshan/integration/tests/SecurityTest.java

```diff
@@ -286,7 +286,7 @@ public void registered_device_with_x509cert_to_server_with_x509cert() throws Non
         helper.client.start();
         helper.waitForRegistration(1);
 
-        assertEquals(1, helper.server.getClientRegistry().allClients().size());
+        assertEquals(1, helper.server.getRegService().getAllRegistrations().size());
```

Comment:
I wonder if we should try to be consistent here, one time we use "Reg" and another time "Registrations"